### PR TITLE
Bubble comments

### PR DESCRIPTION
**Pull request by SansPseudoFix**
**This is a proposal**

How it looks like:

comic_comments11

comic_comments12

### CODE REVIEW

File: app/assets/stylesheets/comments.scss [hunk 1]
```diff
@@ -29,7 +55,8 @@
     }
   }
   .submit_button {
-    margin-top: 10px;
+    margin-top: 5px;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Properties should be ordered display, margin-top, padding-left

File: app/assets/stylesheets/comments.scss [hunk 2]
```diff
@@ -37,11 +64,15 @@
     display: none;
   }
   .comment_box {
+    width: 100%;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Properties should be ordered border, height, resize, width

File: app/assets/stylesheets/comments.scss [hunk 3]
```diff
@@ -1,26 +1,52 @@
...
+      width: 0;
+    }
+
     .avatar {
       height: 35px;
       width: 35px;
+      margin-right: 10px;
+    }
+
+    .bd {
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Selector should have depth of applicability no greater than 3, but was 4

File: app/assets/stylesheets/comments.scss [hunk 4]
```diff
@@ -1,26 +1,52 @@
...
     }
     .media { margin-top: 10px; }
   }
+
+  .comments > .comment {
+    margin: -13px 0;
+    padding: 3px 0;
+  }
+
+  .comment.new_comment_form_wrapper {
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Selector `new_comment_form_wrapper` should be written in lowercase with hyphens

File: app/assets/stylesheets/comments.scss [hunk 5]
```diff
@@ -37,11 +64,15 @@
...
   .comment_box {
+    width: 100%;
     height: 35px;
     resize: none;
+    border: 0;
   }
+  
   textarea.comment_box:focus, textarea.comment_box:valid, textarea.comment_box:active {
-    border-color: $border-dark-grey;
+    border-bottom: 1px solid $border-grey;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Each selector in a comma sequence should be on its own single line

File: app/assets/stylesheets/comments.scss [hunk 6]
```diff
@@ -37,11 +64,15 @@
     display: none;
   }
   .comment_box {
+    width: 100%;
     height: 35px;
     resize: none;
+    border: 0;
   }
+
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Line contains trailing whitespace

File: app/assets/stylesheets/comments.scss [hunk 7]
```diff
@@ -1,26 +1,54 @@
...
       width: 35px;
+      margin-right: 10px;
+    }
+
+    .bd {
+      box-shadow: 0 0 1px 1px $light-grey;
+      margin-right: 10px;
+      padding: 5px;
+
+      .collapsible { width: 100%;}
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Nesting should be no greater than 3, but was 4

Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Selector should have depth of applicability no greater than 3, but was 5

File: app/assets/stylesheets/comments.scss [hunk 8]
```diff
@@ -37,11 +66,15 @@
...
   .comment_box {
+    width: 100%;
     height: 35px;
     resize: none;
+    border: 0;
   }
+
   textarea.comment_box:focus, textarea.comment_box:valid, textarea.comment_box:active {
-    border-color: $border-dark-grey;
+    border-bottom: 1px solid $border-grey;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/comments.scss]: Each selector in a comma sequence should be on its own single line

**Comment by SansPseudoFix** [comment]
Mh. I thought I solved this pront issues...

**Comment by AugierLe42e** [comment]
Like it !

**Comment by Realtin** [comment]
I'm not 100% sure about this. While it does look good, it might be contrasting to the very sleek and rather serious new design on develop.
What do the others think?

**Comment by AugierLe42e** [comment]
The comments are clearly separated from each other so clearly easier to read. That's all that matters to me.

**Comment by AugierLe42e** [comment]
@SansPseudoFix, try this:

```
diff --git a/app/assets/stylesheets/comments.scss b/app/assets/stylesheets/comments.scss
index 6746662..9d7fdc3 100644
--- a/app/assets/stylesheets/comments.scss
+++ b/app/assets/stylesheets/comments.scss
@@ -19,15 +19,15 @@
 
   .comments > .comment, .comment.new_comment_form_wrapper {
     &:before {
-      border-bottom: 11px solid transparent;
-      border-right: 14px solid $light-grey;
-      border-top: 11px solid transparent;
+      border-bottom: 10px solid transparent;
+      border-right: 16px solid $light-grey;
+      border-top: 10px solid transparent;
       content: '';
       display: block;
       height: 0;
-      left: 41px;
+      left: 39px;
       position: relative;
-      top: 25px;
+      top: 27px;
       width: 0;
     }
 
@@ -47,7 +47,6 @@
 
     .info {
       margin-top: 5px;
-      font-size: normal;
       line-height: normal;
     }
 
@@ -56,13 +55,14 @@
       padding-left: 3px;
     }
   }
+
+  .comment.new_comment_form_wrapper > .bd {
+    padding: 0;
+    .submit_button { margin: 5px; }
+  }
+
   .submit_button {
     margin-top: 5px;
-
-    input {
-      float: right;
-    }
-    padding-left: 12px;
     display: none;
   }
   .comment_box {
diff --git a/app/assets/templates/comment-stream_tpl.jst.hbs b/app/assets/templates/comment-stream_tpl.jst.hbs
index ac8f118..8ee8c0d 100644
--- a/app/assets/templates/comment-stream_tpl.jst.hbs
+++ b/app/assets/templates/comment-stream_tpl.jst.hbs
@@ -20,9 +20,10 @@
 
     <div class="bd">
       <form accept-charset="UTF-8" action="/posts/{{id}}/comments" class="new_comment" id="new_comment_on_{{id}}" method="post">
-        <textarea class="comment_box form-control" id="comment_text_on_{{id}}" name="text" rows="1" required placeholder="{{t "stream.comment"}}" />
-        <div class="submit_button">
-          <input class="btn btn-primary" id="comment_submit_{{id}}" name="commit" type="submit" value="{{t "stream.comment"}}" />
+        <textarea class="comment_box form-control" id="comment_text_on_{{id}}"
+                  name="text" rows="1" required placeholder="{{t "stream.comment"}}"></textarea>
+        <div class="submit_button clearfix">
+          <input class="btn btn-primary pull-right" id="comment_submit_{{id}}" name="commit" type="submit" value="{{t "stream.comment"}}" />
         </div>
       </form>
     </div>
diff --git a/app/assets/templates/comment_tpl.jst.hbs b/app/assets/templates/comment_tpl.jst.hbs
index 142661c..e4cb002 100644
--- a/app/assets/templates/comment_tpl.jst.hbs
+++ b/app/assets/templates/comment_tpl.jst.hbs
@@ -11,7 +11,7 @@
       {{#if canRemove}}
         <a href="#" class="delete comment_delete" title="{{t "delete"}}">
           <i class="entypo-trash"></i>
-        <a/>
+        </a>
       {{else}}
         <a href="#" data-type="comment" class="comment_report" title="{{t "report.name"}}">
           <i class="entypo-warning"></i>
@@ -26,14 +26,12 @@
       -
     <span class="info">
       <a href="/posts/{{parent.id}}#{{guid}}" class="permalink_comment">
-        <time class="timeago" data-original-title="{{{localTime created_at}}}" datetime="{{created_at}}"/>
+        <time class="timeago" data-original-title="{{{localTime created_at}}}" datetime="{{created_at}}"></time>
       </a>
     </span>
 
     <div class="collapsible comment-content markdown-content">
       {{{text}}}
     </div>
-
-
   </div>
 </div>
```

**Comment by svbergerem** [comment]
I agree with @Realtin.

**Comment by denschub** [comment]
Closing due to excessive inactivity. Feel free to reopen if you want to continue working on this, but this probably needs a loomio decision.

**Comment by SansPseudoFix** [comment]
I don't want to *fight* for it, so, yup, you can close it.

**Comment by Flaburgan** [comment]
@SansPseudoFix maybe it's time to have a look at the theme system of diaspora*? :)
Waiting for the front code to be stabilized enough before 0.6 of course, or this would constantly break I guess.

**Pull request closed without merging**